### PR TITLE
new challenge about interaction failures in mobile app

### PR DESCRIPTION
**Pull request by acherm**
SPLC 2024

### CODE REVIEW

File: 2024/interaction_failures_mobileapp/.index.html.~undo-tree~ [hunk 1]
```diff
@@ -0,0 +1,15 @@
+(undo-tree-save-format-version . 1)
```
Comment (tthuem) [inline 2024/interaction_failures_mobileapp/.index.html.~undo-tree~:1]: please avoid adding this file

**Review by tthuem — CHANGES_REQUESTED** [review]
see detailed comment

**Review by eulerhm — CHANGES_REQUESTED** [review]
Can you please remove the hyphens from the abstract?

**Comment by tthuem** [comment]
@jabiercoding @garganti @xdevroey From the history it seems that you all have rights to apply changes to the repository. Could you merge the PR or grant us (@acherm @tthuem) write access to apply changes on our own?

**Comment by tthuem** [comment]
@acherm Will you revise the commits based on the feedback or shall I help out?

**Comment by xdevroey** [comment]
I do not have admin rights on the repo. I cannot grant you write access. Maybe @jabiercoding, you can add us (past chairs) all in the organization variability-challenges so we can easily pass on the next chairs (and keep track of who has been chairing the track previously)?

**Comment by garganti** [comment]
While we are investigating who can grant access to whom, I have approved the PR

**Comment by jabiercoding** [comment]
Hello all, thanks @xdevroey and @garganti , the variability-challenges was a personal account so I now changed it to an organization and gave owner rights to @acherm @tthuem .

**Comment by tthuem** [comment]
Many thanks!

**Pull request merged**